ui/partials/perf/graphsctrl.html

```diff
@@ -32,6 +36,12 @@
     <div id="data-display">
       <select ng-model="myTimerange" ng-options="timerange.text for timerange in timeranges track by timerange.value" ng-change="timeRangeChanged()" ng-disabled="seriesList.length == 0">
       </select>
+      <select ng-model="myMeasure" ng-change="myMeasureChanged()">
+        <option value="mean">Mean</option>
```

Comment:
Could you put Median after Mean? That's the one that people are most likely to switch to.